docs/references/protocol/ledger-data/ledger-entry-types/directorynode.md

```diff
@@ -43,37 +45,46 @@ There are three kinds of directory:
 ```json
 {
     "Flags": 0,
+    "IndexNext": "0",
+    "IndexPrevious": "0",
     "Indexes": [
-        "AD7EAE148287EF12D213A251015F86E6D4BD34B3C4A0A1ED9A17198373F908AD",
-        "E83BBB58949A8303DF07172B16FB8EFBA66B9191F3836EC27A4568ED5997BAC5"
+        "1192C0191D1B8861AA6F5A84A575E0CBE4B97574A5E8B3D7B7AD64643EE38CA7",
+        "16A0674079229DB47EDDF4FD83AFEA59ADAC944DD5F16EA5D9C989ED8F918AE0",
+        "1F65776E640C97B76E365763E97E5B59B6C4CDBB46FB7C8869D1712528985E6D",
+        "35D6A9F578E63C875EDB6348E55EFADBD300A0817290276D8CC3DD3587FAD4B3",
+        "36B236D80688C2975A5D24935020B75BEB4B26F5115D71943356E86CCD3B8CE4",
+        "39E8F12D519E5C6C1AC36434D7340281C362508B7D5BC863166C8FE8621A124C",
+        "4DF14053E1BD697C5B4A4A1A7BA8988BD802F0CD5DB6ED9C2AC74AD8A7B91A35",
+        "5E2D97ABAB0D2BE1948F275823096597E3359DD0696CF2938A712169394236BE",
+        "678CE03A2F8157FBF7D5EFDED2D55D127F60EC26BC4F51DBC8FB05DF370B248E",
+        "8250CE37F6495903C1F7D16E072E8823ECE06FA73F011A0F8D79D5626BF581BB",
+        "C353DA9F84EB02B4206D6F5166A9277916559115EEDF7B841C38E4473084A010",
+        "CB2D979DE863A7AF792A12D6C4518E2B299EF782E361705DE7F1D0077521D521",
+        "DFA7CB434A3D9D782C2FACEB95F431476D3AAAD62078C0FBE8C115E00039C6F5"
     ],
     "LedgerEntryType": "DirectoryNode",
-    "Owner": "rpR95n1iFkTqpoy1e878f4Z1pVHVtWKMNQ",
-    "RootIndex": "193C591BF62482468422313F9D3274B5927CA80B4DD3707E42015DD609E39C94",
-    "index": "193C591BF62482468422313F9D3274B5927CA80B4DD3707E42015DD609E39C94"
+    "Owner": "rBTwLga3i2gz3doX6Gva3MgEV8ZCD8jjah",
+    "PreviousTxnID": "CB802FC111C4C03B1E1D762E813D3F1F47347E57C68D00B5F92822C417C2484C",
+    "PreviousTxnLgrSeq": 91448329,
+    "RootIndex": "0A2600D85F8309FE7F75A490C19613F1CE0C37483B856DB69B8140154C2335F3",
+    "index": "0A2600D85F8309FE7F75A490C19613F1CE0C37483B856DB69B8140154C2335F3"
 }
 ```
 {% /tab %}
 
 {% tab label="NFT Offer Directory" %}
 ```json
 {
-   "result": {
```

Comment:
The previous result included some wrapping JSON from an API request. The updated version matches the other examples by showing just the ledger entry itself.